src/Draco.Editor.Web/app/ts/app.ts

```diff
@@ -0,0 +1,120 @@
+import * as monaco from 'monaco-editor';
+// @ts-ignore
+import editorWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
+// @ts-ignore
+import jsonWorker from 'monaco-editor/esm/vs/language/json/json.worker?worker';
+// @ts-ignore
+import cssWorker from 'monaco-editor/esm/vs/language/css/css.worker?worker';
+// @ts-ignore
+import htmlWorker from 'monaco-editor/esm/vs/language/html/html.worker?worker';
+// @ts-ignore
+import tsWorker from 'monaco-editor/esm/vs/language/typescript/ts.worker?worker';
+import { loadWASM } from 'onigasm'; // peer dependency of 'monaco-textmate'
+import { Registry } from 'monaco-textmate'; // peer dependency
+import { wireTmGrammars } from 'monaco-editor-textmate';
+// @ts-ignore
+import onigasm from 'onigasm/lib/onigasm.wasm?url';
+// @ts-ignore
+import vsthemeRaw from '../data/vscode.converted.theme.json?raw';
+import JSON5 from 'json5';
+import tslib from 'tslib';
+
+declare global {
+    class DotNet {
+        static invokeMethodAsync<T>(assemblyName: string, methodIdentifier: string, ...args: any[]): Promise<T>
+    }
+}
+
+// hack for vite bundler...
+self.MonacoEnvironment = {
+    getWorker(_, label) {
+        if (label === 'json') {
+            return new jsonWorker()
+        }
+        if (label === 'css' || label === 'scss' || label === 'less') {
+            return new cssWorker()
+        }
+        if (label === 'html' || label === 'handlebars' || label === 'razor') {
+            return new htmlWorker()
+        }
+        if (label === 'typescript' || label === 'javascript') {
+            return new tsWorker()
+        }
+        return new editorWorker()
```

Comment:
I'm not a TS expert but doesn't it have a switch?

Reply:
It's a copy paste from a GitHub issue.
Workaround due to the bundler, will be removed soon.

Reply:
Removed most of it anyway, as we don't need workers of these languages.